kafka/consumer/group.py

```diff
@@ -861,6 +861,48 @@ def metrics(self, raw=False):
             metrics[k.group][k.name] = v.value()
         return metrics
 
+    def offsets_for_times(self, timestamps):
+        """
+        Look up the offsets for the given partitions by timestamp. The returned
```

Comment:
timestamps is a map `{TopicPartition: int}`. The wording was taken from https://kafka.apache.org/0110/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html#offsetsForTimes(java.util.Map), I think it's OK...